packages/runtime/runtime-utils/src/summarizerNode/summarizerNodeWithGc.ts

```diff
@@ -165,7 +175,9 @@ export class SummarizerNodeWithGC extends SummarizerNode implements IRootSummari
 
         // If trackState is true, get summary from base summarizer node which tracks summary state.
         // If trackState is false, get summary from summarizeInternal.
-        return trackState ? super.summarize(fullTree) : this.summarizeFn(fullTree, trackState);
+        return trackState
+            ? super.summarize(fullTree, true, telemetryContext)
```

Comment:
code is more readable if it looks like (fullTree, true /* name of argument */, telemetryContext)